### PR TITLE
update placement-rules doc about max-replica

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Ref https://github.com/tikv/pd/pull/3163

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: configure-placement-rules.md [hunk 1]
```diff
@@ -111,7 +111,8 @@ PD 同样将根据系统的 `max-replicas` 及 `location-labels` 生成默认的
 
 > **注意：**
 >
-> 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> 开启 Placement Rules 后，且有多条 rule 的情况下，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> 如果开启 Placement Rules 但只有一条 rule，当尝试改变 `max-replicas` 及 `location-labels` 时，系统会自动更新默认的 rule。
```
Comment (qiancai) [inline configure-placement-rules.md:115]:
```suggestion
> - 开启 Placement Rules 且创建了多条 rule 的情况下，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
> - 开启 Placement Rules 且只创建了一条 rule 的情况下，当改变 `max-replicas` 及 `location-labels` 配置项时，系统会自动更新这条 rule。
```

Comment (rleungx) [reply configure-placement-rules.md:115]: 创建 -> 存在？

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.3 label because the v7.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.3 and will no longer receive new updates

File: tiflash/troubleshoot-tiflash.md [hunk 2]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 
     > **注意：**
     >
-    > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 Placement Rules 且存在多条 rule 的情况下，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 Placement Rules 且只存在一条默认的 rule 的情况下，当改变 `max-replicas` 及 `location-labels` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (qiancai) [inline tiflash/troubleshoot-tiflash.md:79]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 `max-replicas` 及 `location-labels` 配置项时，系统会自动更新这条默认的 rule。
```

File: configure-placement-rules.md [hunk 3]
```diff
@@ -111,7 +111,8 @@ PD 同样将根据系统的 `max-replicas` 及 `location-labels` 生成默认的
 
 > **注意：**
 >
-> 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> - 开启 Placement Rules 且存在多条 rule 的情况下，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> - 开启 Placement Rules 且只存在一条默认的 rule 的情况下，当改变 `max-replicas` 及 `location-labels` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (rleungx) [inline configure-placement-rules.md:115]: isolation level 也加上？

File: tiflash/troubleshoot-tiflash.md [hunk 4]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 
     > **注意：**
     >
-    > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
```
Comment (qiancai) [inline tiflash/troubleshoot-tiflash.md:78]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
```

File: tiflash/troubleshoot-tiflash.md [hunk 5]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
...
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (qiancai) [inline tiflash/troubleshoot-tiflash.md:79]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 及 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```

**Review by rleungx — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: configure-placement-rules.md [hunk 6]
```diff
@@ -107,11 +107,12 @@ enable-placement-rules = true
...
 ```
 
-PD 同样将根据系统的 `max-replicas` 及 `location-labels` 生成默认的规则。
+PD 同样将根据系统的 `max-replicas`、`location-labels` 及 `isolation-level` 生成默认的规则。
 
 > **注意：**
 >
-> 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> - 开启 Placement Rules 且存在多条 rule 的情况下，原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+> - 开启 Placement Rules 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 及 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (hfxsd) [inline configure-placement-rules.md:115]: 这里的 及 要改成 或吗？

File: configure-placement-rules.md [hunk 7]
```diff
@@ -125,7 +126,7 @@ pd-ctl config placement-rules disable
 
 > **注意：**
 >
-> 关闭 Placement Rules 后，PD 将使用原先的 `max-replicas` 及 `location-labels` 配置。在 Placement Rules 开启期间对 Rule 的修改不会导致这两项配置的同步更新。此外，设置好的所有 Rule 都会保留在系统中，会在下次开启 Placement Rules 时被使用。
+> 关闭 Placement Rules 后，PD 将使用原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置。在 Placement Rules 开启期间对 Rule 的修改不会导致这两项配置的同步更新。此外，设置好的所有 Rule 都会保留在系统中，会在下次开启 Placement Rules 时被使用。
```
Comment (hfxsd) [inline configure-placement-rules.md:129]:
```suggestion
> 关闭 Placement Rules 后，PD 将使用原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置。在 Placement Rules 开启期间对 Rule 的修改不会导致这三项配置的同步更新。此外，设置好的所有 Rule 都会保留在系统中，会在下次开启 Placement Rules 时被使用。
```

File: tiflash/troubleshoot-tiflash.md [hunk 8]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 
     > **注意：**
     >
-    > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 及 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (hfxsd) [inline tiflash/troubleshoot-tiflash.md:79]: 这里的 及 要改成 或 吗？

File: configure-placement-rules.md [hunk 9]
```diff
@@ -82,7 +82,7 @@ Placement Rules 特性在 TiDB v5.0 及以上的版本中默认开启。如需
 enable-placement-rules = true
 ```
 
-这样，PD 在初始化成功后会开启这个特性，并根据 `max-replicas` 及 `location-labels` 配置生成对应的规则：
+这样，PD 在初始化成功后会开启这个特性，并根据 `max-replicas`、`location-labels` 及 `isolation-level` 配置生成对应的规则：
```
Comment (hfxsd) [inline configure-placement-rules.md:85]:
```suggestion
这样，PD 在初始化成功后会开启这个特性，并根据 [`max-replicas`](pd-configuration-file.md#max-replicas)、`location-labels` 及 `isolation-level` 配置生成对应的规则：
```

Comment (qiancai) [inline configure-placement-rules.md:85]:
```suggestion
这样，PD 在初始化成功后会开启这个特性，并根据 [`max-replicas`](/pd-configuration-file.md#max-replicas)、[`location-labels`](/pd-configuration-file.md#location-labels) 及 [`isolation-level`](/pd-configuration-file.md#isolation-level) 配置生成对应的规则：
```

File: tiflash/troubleshoot-tiflash.md [hunk 10]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 
     > **注意：**
     >
-    > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 `max-replicas`、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
```
Comment (hfxsd) [inline tiflash/troubleshoot-tiflash.md:78]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 [`max-replicas`](/pd-configuration-file.md#max-replicas)、`location-labels` 及 `isolation-level` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
```

**Review by hfxsd — APPROVED** [review]

File: tiflash/troubleshoot-tiflash.md (hunk 10 above)
Comment (qiancai) [inline tiflash/troubleshoot-tiflash.md:78]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 [`max-replicas`](/pd-configuration-file.md#max-replicas)、[`location-labels`](/pd-configuration-file.md#location-labels) 及 [`isolation-level`](/pd-configuration-file.md#isolation-level) 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
```

File: configure-placement-rules.md (hunk 6 above)
Comment (qiancai) [inline configure-placement-rules.md:115]:
```suggestion
> - 开启 Placement Rules 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 或 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```

File: tiflash/troubleshoot-tiflash.md [hunk 11]
```diff
@@ -75,7 +75,8 @@ aliases: ['/docs-cn/dev/tiflash/troubleshoot-tiflash/','/docs-cn/dev/tiflash/tif
 
     > **注意：**
     >
-    > 开启 Placement Rules 后，原先的 `max-replicas` 及 `location-labels` 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且存在多条 rule 的情况下，原先的 [`max-replicas`](/pd-configuration-file.md#max-replicas)、[`location-labels`](/pd-configuration-file.md#location-labels) 及 [`isolation-level`](/pd-configuration-file.md#isolation-level) 配置项将不再生效。如果需要调整副本策略，应当使用 Placement Rules 相关接口。
+    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 及 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```
Comment (qiancai) [inline tiflash/troubleshoot-tiflash.md:79]:
```suggestion
    > - 开启 [Placement Rules](/configure-placement-rules.md) 且只存在一条默认的 rule 的情况下，当改变 `max-replicas`、`location-labels` 或 `isolation-level` 配置项时，系统会自动更新这条默认的 rule。
```

**Pull request merged**